### ISSUE TITLE
spindle pole body review

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#1320]
I thought I hade done the kinetochore ones....
kinetochore
 - [x] SPBC11C11.03	ndc80	NMS complex subunit Ndc80
 - [x] SPBC336.08	spc24	NMS complex subunit Spc24 
 - [x]  SPAC688.02c	mis14	NMS complex subunit Mis14 
 - [x]  SPCC1020.02	spc7	NMS complex subunit, Spc105/KNL-1 family member, blinkin, Spc7
 - [x]  SPAC27F1.04c	nuf2	NMS complex subunit Nuf2 
- [x]   SPAC1687.20c	mis6	kinetochore protein, CENP-I ortholog Mis6
- [x]  SPBC776.16	mis20	centromere protein Mis20/Eic2
- [x]   SPBP22H7.09c	mis15	kinetochore protein, CENP-N ortholog Mis15 
- [x]  SPCC576.12c	mhf2	kinetochore protein, CENP-X ortholog, Mhf2
- [x]   SPAC6G9.06c	pcp1	pericentrin Pcp1
- [x]  SPAC25B8.14	mal2	kinetochore protein, CENP-O ortholog Mal2
- [x]  SPBC800.13	cnp20	histone H4 variant, CENP-T ortholog 
- [x] SPAC11H11.05c	fta6	Mis6-Sim4 complex Fta6
- [x]  SPCC188.04c	spc25	NMS complex subunit Spc25 
- [x]  SPBC409.04c	mis12	NMS complex subunit Mis12
- [x]  SPAC664.01c	swi6	HP1 family chromodomain protein Swi6
- [x]  SPCC1235.07	fta7	kinetochore protein, CENP-Q homolog Fta7 
- [x]  SPAC9E9.10c	cbh1	kinetochore protein, CENP-B homolog Cbh1 

DASH (is at the interface between spindle and kinetochore...check)
SPAC1805.07c	dad2	DASH complex subunit Dad2
SPAC8C9.17c	spc34	DASH complex subunit Spc34
SPBC27.02c	ask1	DASH complex subunit Ask1
SPAC16A10.05c	dad1	DASH complex subunit Dad1 
SPAC589.08c	dam1	DASH complex subunit Dam1 

nuclear pore
- [x]  SPCC290.03c	nup186	nucleoporin Nup186
- [x]  SPBC17G9.04c	nup85	nucleoporin Nup85 

likely kinetochore (or centomere pre k- assembly)
- [x]  SPBC211.02c	cwf3	complexed with Cdc5 protein Cwf3
- [x]  SPCC1393.05	ers1	RNA-silencing factor Ers1
- [x]  SPCC11E10.08	rik1	silencing protein Rik1 
- [x]  SPBC244.02c	utp6	U3 snoRNP-associated protein Utp6
- [x]  SPBC23E6.04c	utp10	U3 snoRNP-associated protein Utp10
- [x]  SPCC1672.07	utp21	U3 snoRNP-associated protein Utp21
- [x]  SPBC3B9.02c	cwf28	splicing factor Cwf28
- [x]  SPCC188.11	prp45	splicing factor Prp45
- [x]  SPAC9.13c	cwf16	splicing factor Cwf16 
- [x] SPCC364.02c	bis1	splicing factor Bis1 
- [x] SPAC1486.03c	SPAC1486.03c	RNA-binding splicing factor (predicted) 
- [x] SPAC4A8.09c	cwf21	complexed with Cdc5 protein Cwf21
- [x] SPAC29E6.02	prp3	U4/U6 x U5 tri-snRNP complex subunit Prp3 
- [x] SPAC22F3.11c	snu23	U4/U6 x U5 tri-snRNP complex subunit Snu23 

Still more to check, but a lot might also localize to SPB, probably need more info for those...

**Comment by ValWood** [comment]
to check

abp2 , acr1 , alp14 , alp16 , apl1 , apm1 , apm3 , apm4 , arz1 , bet4 , bqt2 , cam1 , cdc11 , cdc16 , cdc31 , cdr2 , cfh1 , chp1 , chp2 , cig2 , ckb1 , clr4 , crb3 , csi1 , cut20 , cut23 , cut7 , cwf16 , dbl1 , dbl2 , dbl5 , dfp1 , dlc1 , dsk1 , ebp2 , epe1 , est1 , fta6 , gef2 , get2 , gfh1 , gid7 , git1 , gpa2 , gtb1 , hus1 , ipi1 , irs4 , klp5 , klp6 , klp8 , kms1 , kms2 , kri1 , las1 , lsb5 , mat3-Mc , mcp6 , mde4 , mit1 , mlh1 , mob1 , mod21 , mph1 , mto1 , mug30 , mug62 , mug93 , nog2 , nse5 , ntr1 , oca3 , pac2 , pci2 , pcp1 , peg1 , pmk1 , pmo25 , pmt3 , ppc89 , ppk15 , ppk21 , ppk9 , prp11 , psl1 , rad16 , rad24 , rad4 , rct1 , rhp7 , rim20 , rix1 , rng2 , rpc31 , rpc82 , rrn11 , rrp12 , rrp15 , rrp4 , rrp41 , rsp1 , rtp1 , sec10 , sec6 , set9 , sfi1 , shf1 , sid2 , slp1 , smc5 , snt2 , sog2 , sol1 , spb1 , spg1 , spk1 , spt16 , sqs2 , srs2 , ssm4 , ssp2 , ste7 , swi1 , tas3 , tcb2 , tfg2 , tgs1 , tpr1 , trs120 , urk1 , utp502 , wee1 , SPAC11D3.16c , SPAC1687.14c , SPAC323.03c , SPAC3H5.08c , SPBC1604.16c , SPBC1685.04 , SPBC28E12.02 , SPBC31F10.10c , SPBC56F2.03 , SPCC16C4.02c

**Comment by ValWood** [comment]
remove
SPBC28E12.02

keep for now
SPBC31F10.10c , SPBC56F2.03 , SPCC16C4.02c

**Comment by ValWood** [comment]
query 
nse5
smc5 (plus other annotations)

**Comment by ValWood** [comment]
- [ ] pmt3, check, also Tanaka paper, probably "colicalizes"

**Comment by ValWood** [comment]
I have removed quite a lot of false positive spindle associations.
This was a "co-localizagtion study" and over time it has become clear that a lot of these are actually "kinetochore comonents". I therefore removed (added to the corresponding 'deleted file")
anything related to splicing, RNA processing, kinetochore or centromeric chromatin assemlby.

I removed a  few general nuclear periphery ones too, which would also have partial co-localization wth SPB.

For now, I retained the double-strand break localizing ones....however I suspect this is due to the localization of  double strand breaks to the nuclear envelope in the vacinity of the SPB too... I will find out more about  this. So for this round I am done here with the 'high priority' part. I'll keep this ticket open for now.

**Comment by ValWood** [comment]
for the record, I upped this based on
https://www.ncbi.nlm.nih.gov/pubmed/28356353

In budding yeast (Saccharomyces cerevisiae) the multi-layered spindle pole body (SPB) is embedded in the nuclear envelope (NE) at fusion sites of the inner and outer nuclear membrane. The SPB is built from 18 different proteins, including the three integral membrane proteins Mps3, Ndc1, and Mps2.

Ours has 211 (mitotic).
However, I think its fine. Lots of signalling molecules localize to the SPB  transiently...even if there are fewer structural components.

Anyway I reduced somewhat, can refine further based on outstanding questions above (some of these could also be meiotic, since they were expressed out of context...I kept anything I was unsure about for now)
